docs/Reference/Consensus/IBFT-RPC-API.md

```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
 - `hash`: `String` - The retrieved block's hash
 - `author`: `String` - The address of the block proposer
 - `committers`: `[]String` - The list of all addresses whose seal appears in this block
+
+### istanbul.status
+
+Returns the signing status of blocks for a given block range. If start block number and end block number are not provided then it returns the status of last 64 blocks by default.
+
+```js
+istanbul.status(startBlockNumber,endBlockNumber)
+```
+
+#### Parameters
+
+- `Number` - start block number
+- `Number` - end block number
+
+#### Returns
+
+`Object`
+
+- `numBlocks`: `Number` - no of blocks for which sealer activity is retrieved
+- `sealerActivity`: `map[string]Number` - key is the validator and value is the number of blocks sealed by the validator
+
+### istanbul.isValidator
+
+Returns true if this node was the validator for the given blockNumber. If blockNumber is not provided it takes the latest block number by default.
```

Comment:
- "`true`" and "`blockNumber`" should be surrounded by backticks (`).
- Change "was" to "is".

Reply:
done